vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -396,7 +397,6 @@ protected String getInternalId() {
          *
          * @return the width of this column as a CSS-string
```

Comment:
Initial might not be good here since if you change width from server-side during life time of the Grid, this method will return that new server-side width instead of the initial width.

Added a note to the javadoc.